test/e2e/pd.go

```diff
@@ -75,7 +75,7 @@ var _ = framework.KubeDescribe("Pod Disks", func() {
 	})
 
 	// Flaky-- Issue #27691
-	It("[Flaky] should schedule a pod w/ a RW PD, remove it, then schedule it on another host [Slow]", func() {
+	It("[Flaky] should schedule a pod w/ a RW PD, ungracefully remove it, then schedule it on another host [Slow]", func() {
```

Comment:
I guess it's too late to comment on this pr, but I'm curious to know what you think. It seems that the defer below should be moved to after the call to podClient.Create(). If the Create() fails then the not-yet-created pod will be try to be deleted, and the not-yet-attached  disks will try to be detached. This is probably very minor, or perhaps I'm missing something?